test/test_files/extension/extension.test

```diff
@@ -36,3 +36,10 @@ Binder exception: The extension sqlitescanner is neither an official extension,
 -STATEMENT LOAD EXTENSION '/tmp/iceberg';
 ---- error
 Binder exception: The extension /tmp/iceberg is neither an official extension, nor does the extension path: '/tmp/iceberg' exists.
+
+-CASE LoadNotInstalledExtension
+-SKIP
```

Comment:
Is this meant to be skipped?

Reply:
Yes, we have to introduce uninstall extension before the test can be enabled.
Some CIs have the extension installed.